spec.md

```diff
@@ -33,6 +33,7 @@ The goal of this specification is to standardize container image distribution ba
   - [Listing Repositories](#listing-repositories)
```

Comment:
maybe we hold off on mods to spec.md until the reorg is merged: https://github.com/opencontainers/distribution-spec/issues/163